pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -185,6 +199,20 @@ func (o createAuthInfoOptions) validate() error {
 	if len(methods) > 1 {
 		return fmt.Errorf("You cannot specify more than one authentication method at the same time: %v", strings.Join(methods, ", "))
 	}
+	if o.embedCertData.Value() {
+		if o.clientCertificate.Value() == "" {
+			return errors.New("You must specify a client certificate to embed")
+		}
+		if o.clientKey.Value() == "" {
```

Comment:
Specifying just a cert would be more common, in the case where they had an existing cert/key, and were updating the cert, while keeping their current key. The reverse would be less common.